EpikCord/__init__.py

```diff
@@ -1156,6 +1159,23 @@ async def command_error(
     ):
         logger.exception(error)
 
+    async def auto_moderation_rule_create(self, data:dict) -> AutoModerationRule:
+        rule = AutoModerationRule(self, data)
+        self.auto_moderation_cache.add_to_cache(rule.guild_id, rule)
+        return rule
+    
+    async def auto_moderation_rule_update(self, data:dict)-> tuple[Optional[AutoModerationRule], AutoModerationRule]:
```

Comment:
`tuple[Optional[AutoModerationRule], AutoModerationRule]`
is just way too long, need alias for this